pkg/api/probe.go

```diff
@@ -7,22 +7,12 @@ package api
 import "sync/atomic"
 
 // ProbeStatus is the status of a probe.
-// ProbeStatus is treated as a sync/atomic int32.
-type ProbeStatus int32
-
-// get returns the value of the ProbeStatus.
-func (ps *ProbeStatus) get() ProbeStatus {
-	return ProbeStatus(atomic.LoadInt32((*int32)(ps)))
-}
-
-// set updates the value of the ProbeStatus.
-func (ps *ProbeStatus) set(v ProbeStatus) {
-	atomic.StoreInt32((*int32)(ps), int32(v))
-}
+// ProbeStatus is treated as a sync/atomic bool.
+type ProbeStatus bool
 
 // String implements the fmt.Stringer interface.
 func (ps ProbeStatus) String() string {
-	switch ps.get() {
+	switch ps {
```

Comment:
since `ProbeStatus` type is `bool` then `uknown` string value is not needed anymore (this is unreachable code), right?